app/src/pages/admin/serverSettingsPage/serverSettingsTabs/authConfigurationTab/forms/inactivityTimeoutForm/inactivityTimeoutForm.jsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2025 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import classNames from 'classnames/bind';
+import { SectionHeader } from 'components/main/sectionHeader';
+import React, { useState } from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+import { InputDropdown } from 'components/inputs/inputDropdown';
+import { COMMON_LOCALE_KEYS } from 'common/constants/localization';
+import { useDispatch, useSelector } from 'react-redux';
+import { sessionExpirationTimeSelector, updateExpirationSessionAction } from 'controllers/appInfo';
+import { useTracking } from 'react-tracking';
+import { ADMIN_SERVER_SETTINGS_PAGE_EVENTS } from 'components/main/analytics/events';
+import styles from './inactivityTimeoutForm.scss';
+
+const cx = classNames.bind(styles);
+
+const messages = defineMessages({
+  formHeader: {
+    id: 'SessionExpirationForm.formHeader',
+    defaultMessage: 'Inactivity timeout',
+  },
+  label: {
+    id: 'SessionExpirationForm.label',
+    defaultMessage: 'Session inactivity timeout',
+  },
+  description: {
+    id: 'SessionExpirationForm.description',
+    defaultMessage: 'Duration of user inactivity before automatic logout.',
+  },
+});
+
+export const InactivityTimeoutForm = () => {
+  const { formatMessage } = useIntl();
+  const { trackEvent } = useTracking();
+  const expirationTimeoutConfig = useSelector(sessionExpirationTimeSelector);
+  const dispatch = useDispatch();
+
+  const [timeout, setTimeout] = useState(expirationTimeoutConfig);
```

Comment:
I suggest to rename it to `inactivityTimeout` and `setInactivityTimeout` to not be confused with Web API timeouts.